README.md

```diff
@@ -67,6 +67,11 @@ The most accurate guess would be one when both, the file name and the content ar
  - `IsDotFile`
  - `IsImage`
 
+### Language colors and groups
```

Comment:
Great! It would be nice to add this section to the [Topic of Content at the top](https://github.com/go-enry/go-enry/blob/97a26011a9977becde4467c6d41eb8705c2b1df0/README.md#go-enry---) as well.

Reply:
added in 4fb0b4c

Reply:
thanks :)